### PR TITLE
Fixes: https://github.com/gitcoinco/web/issues/1030

### PR DESCRIPTION
**Pull request by bakaoh**
##### Description
Add priority to sort token list

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
Asset js

##### Testing
Yes

##### Refers/Fixes
Fixes: #1030

### CODE REVIEW

**Comment by owocki** [comment]
ux questions:

do we want to move them to the top or just duplicate them at the top? 
should there be a 'bar' between the priority sorted ones and the alphabetical ones, like so?

```
ETH
DAI
==========
A
B
C
```

**Comment by owocki** [comment]
code looks good and sane to me!

**Comment by thelostone-mc** [comment]
@PixelantDesign do you think we should repeat it ?
I'm leaning towards it cause it's makes sense as I scroll through the list but then again user might notice it being redundant

**Comment by PixelantDesign** [comment]
@owocki @thelostone-mc 
It's okay to repeat because the list is very long.

I like this approach:

ETH
DAI
==========
A
B
C

**Comment by thelostone-mc** [comment]
@bakaoh Ah sorry about this ! Could we update the code to let the list unchanged and just use the priority to duplicate the tokens & display them on top of the list ?

**Comment by bakaoh** [comment]
Ok, i will do that on monday

**Comment by bakaoh** [comment]
@thelostone-mc -- i've duplicated the tokens & display them on top of the list as your request ^^

File: app/assets/v2/js/tokens.js [hunk 1]
```diff
@@ -168,40 +168,42 @@ var tokens = function(network_id) {
       {'addr': '0xb581e3a7db80fbaa821ab39342e9cbfd2ce33c23', 'name': 'ARCD', 'decimals': 18},
       {'addr': '0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6', 'name': 'RDN', 'decimals': 18},
       {'addr': '0x056017c55aE7AE32d12AeF7C679dF83A85ca75Ff', 'name': 'WYV', 'decimals': 18},
-      {'addr': '0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359', 'name': 'DAI', 'decimals': 18},
+      {'addr': '0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359', 'name': 'DAI', 'decimals': 18, 'priority': 100}
```
Comment (thelostone-mc) [inline app/assets/v2/js/tokens.js]: `, 'priority': 100},`

**Comment by thelostone-mc** [comment]
@PixelantDesign this is how it looks !

It looks off as the when I open the dropdown -> I have to scroll to the top to see DAI and ETH.
Still on 2 minds if it's needed ! @owocki 

Also I believe @PixelantDesign wanted a line separating the two ( a`border-bottom maybe` ? )  as mentioned [above](https://github.com/gitcoinco/web/pull/1115#issuecomment-388233248)

x1

cc @mbeacom

File: app/assets/v2/js/tokens.js (hunk 1 above)
Comment (owocki) [reply app/assets/v2/js/tokens.js]: tokens.js:172 Uncaught SyntaxError: Unexpected token {

**Comment by bakaoh** [comment]
@owocki --- i've done as your requests

**Comment by thelostone-mc** [comment]
@bakaoh could you resolve the conflicts ?
cc @PixelantDesign

**Pull request merged**